GitUI/UserControls/BlameControl.cs

```diff
@@ -323,38 +323,38 @@ private void ProcessBlame(string filename, GitRevision revision, IReadOnlyList<O
             CommitInfo.SetRevisionWithChildren(revision, children);
 
             controlToMask?.UnMask();
+        }
 
-            void BuildAuthorLine(GitBlameLine line)
+        private void BuildAuthorLine(GitBlameLine line, StringBuilder lineBuilder, string dateTimeFormat, string filename)
```

Comment:
To make this more testable (e.g. less dependent on the shared state - `AppSettings`) I recommend passing in all `AppSettings.*` flags.
This way it is easy to add test for all sorts of permutations.